sqlglot/expressions.py

```diff
@@ -5990,6 +5990,10 @@ class DateStrToDate(Func):
     pass
 
 
+class DateStrToTime(Func):
```

Comment:
this already exists, TsOrDsToTime, so let's remove it

Reply:
OK, thanks! I left the tests, since there were no tests for `TsOrDsToTime`.